tests/testAutopas/tests/utils/ArrayMathTest.cpp

```diff
@@ -34,6 +34,32 @@ TEST(ArrayMathTest, testAddDouble) {
   }
 }
 
+TEST(ArrayMathTest, testAddOpInt) {
+  using namespace autopas::utils::ArrayMath::literals;
+  std::array<int, 3> a({1, 2, 3}), b({4, 5, 6});
+  std::array<int, 3> correctResult({5, 7, 9});
+  std::array<int, 3> result = a + b;
+  for (int d = 0; d < 3; ++d) {
+    ASSERT_EQ(result[d], correctResult[d]);
+  }
```

Comment:
I'm not a fan of hardcoded results. It's way more safe, understandable, and resiliant to changes to the test if you calculate the correct results in the test. E.g. like this:
```suggestion
  std::array<int, 3> a({1, 2, 3}), b({4, 5, 6});
  std::array<int, 3> result = a + b;
  for (int d = 0; d < 3; ++d) {
    ASSERT_EQ(result[d], a[d] + b[d]);
  }
```